src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../GSoft.Authentication.ClientCredentialsGrant.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="6.0.16" />
```

Comment:
We'll verify this with the design story but I believe we're going to add this package for JWT validation and claims extraction: [Microsoft.AspNetCore.Authentication.JwtBearer](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.JwtBearer/6.0.0) and it already comes with the whole ASP.NET Core metapackage reference.

If we do use this package for JWT validation we won't need to add this authorization package as it will already be implicitly referenced.

Reply:
I'll remove it for the time being. If require I'll add it in another story.